### PR TITLE
Add "ENV" modifiers for mixbench-hip

### PR DESCRIPTION
**Pull request by acd407**
On my machine, FindHIP.cmake always uses environment variables for ROCM_PATH and HIP_PATH instead of variables. Therefore, it seems that in two places in mixbench-hip, the ENV modifier should be added.

### CODE REVIEW

File: mixbench-hip/CMakeLists.txt [hunk 1]
```diff
@@ -2,14 +2,14 @@ cmake_minimum_required(VERSION 3.10)
 
 if(NOT DEFINED HIP_PATH)
     if(NOT DEFINED ENV{HIP_PATH})
-        set(HIP_PATH "/opt/rocm/hip" CACHE PATH "Path to which HIP has been installed")
```
Comment (ekondis) [inline mixbench-hip/CMakeLists.txt:5]: Can we leave both lines here? Under my environment the variables are the ones that are considered.

File: mixbench-hip/CMakeLists.txt [hunk 2]
```diff
@@ -2,14 +2,14 @@ cmake_minimum_required(VERSION 3.10)
...
+        set(ENV{HIP_PATH} "/opt/rocm/hip")
     else()
         set(HIP_PATH $ENV{HIP_PATH} CACHE PATH "Path to which HIP has been installed")
     endif()
 endif()
 if(NOT DEFINED ROCM_PATH)
     if(NOT DEFINED ENV{ROCM_PATH})
-        set(ROCM_PATH "/opt/rocm/" CACHE PATH "Path to ROCM installation")
```
Comment (ekondis) [inline mixbench-hip/CMakeLists.txt:12]: Same here as above.

**Review by ekondis — CHANGES_REQUESTED** [review]

File: mixbench-hip/CMakeLists.txt (hunk 1 above)
Comment (acd407) [reply mixbench-hip/CMakeLists.txt:5]: in the second commit, set the default value to ENV{HIP_PATH}, the if statement can be deleted.
You can check if it's ok this time.

**Comment by acd407** [comment]
Sorry, I forgot to mention my rocm-hip version. It's 5.7.1-2, from ArchLinux extra repository.

**Pull request merged**